firstvoices/firstvoices/settings.py

```diff
@@ -314,3 +314,13 @@
 # Variables for the environment banners in the admin site
 ENVIRONMENT_NAME = os.getenv("SENTRY_ENVIRONMENT", "Local")
 ENVIRONMENT_COLOR = os.getenv("ENVIRONMENT_COLOR", "#9c9897")
+
+# Variables for the email backend (used in the contact us form)
+ENABLE_SMTP_BACKEND: bool = os.environ.get("ENABLE_SMTP_BACKEND") is not None
+DEFAULT_FROM_EMAIL = os.getenv("DEFAULT_FROM_EMAIL", "user@example.com")
```

Comment:
Two thoughts:

1. I think this value should only have a fallback if smtp is not enabled. If we have an smtp server configured, it is reasonable to let this error out if the default_from_email value is missing.
2. Nothing ever overrides this from_email, so it might be confusing to call it a default. Maybe it could be called email_sender_address to match the other email_ variables?